crates/turborepo-lib/src/run/mod.rs

```diff
@@ -262,8 +262,9 @@ impl Run {
 
         let (sender, receiver) = TuiSender::new();
         let color_config = self.color_config;
+        let repo_root = self.repo_root.clone();
```

Comment:
We should parse the options here and pass them in so we can catch any config/io errors early on